### PR TITLE
Laboratory work #1, Polina Nazhmutdinova - 23FPL1

### PR DESCRIPTION
**Pull request by Palina-frog-dog-whatelse**

### CODE REVIEW

File: lab_1_classify_by_unigrams/main.py [hunk 1]
```diff
@@ -4,7 +4,8 @@
 Language detection
 """
 # pylint:disable=too-many-locals, unused-argument, unused-variable
-
+print("hello world")
```
Comment (artyomtugaryov) [inline lab_1_classify_by_unigrams/main.py:7]: revert

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
Looks like nothing to review. I hope you will be in time

**Comment by artyomtugaryov** [comment]
Nothing to review

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
Nothing to review again

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
THere is not any update

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
No new changes

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
Address previous comments

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
The same comments

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
Nothing new to review

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
Nothing new

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
Nothing to review again

**Comment by artyomtugaryov** [comment]
Nothing to review

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
Nothing to review

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
Nothing new to review

File: lab_1_classify_by_unigrams/main.py (hunk 1 above)
Comment (Palina-frog-dog-whatelse) [reply lab_1_classify_by_unigrams/main.py]: Как мне сделать revert?

**Pull request closed without merging**